src/main/java/net/sf/jabref/logic/layout/LayoutEntry.java

```diff
@@ -358,40 +358,152 @@ private void doOptionField(String s) {
         }
 
     }
-    // added section - end (arudert)
 
-    private static LayoutFormatter getLayoutFormatterByClassName(String className,
-            JournalAbbreviationRepository repostiory)
-            throws Exception {
+    private LayoutFormatter getLayoutFormatterByName(String name) throws Exception {
 
-        if (className.isEmpty()) {
+        if (name.isEmpty()) {
             return null;
```

Comment:
I would return a NotFoundFormatter here as well, avoiding this evil null value :)